docs/best-practices.md

```diff
@@ -132,3 +132,16 @@ the package if they want to experience as smooth upgrade as possible.
 If you're writing an actor for the RHEL 7 to RHEL 8 in-place upgrade workflow and you really need to have some package
 installed, then the only acceptable packages to depend on are the the ones available in the minimal installation of
 RHEL 7 (packages from the @core group + their dependencies).
+
+## When something goes wrong stop the execution of your actor in the right way
+
+If your actor get unexpected input or result of some operation, you must notify the framework about it. For that use these exceptions with a descriptive message:
```

Comment:
This is a bit misleading since only Stop...Error exception takes message, StopActorExecution stops without message

Reply:
I'm not sure if 'you must notify' is the right way of putting it, I would rather say something like If you encounter unexpected input or any other error during execution of an actor and want to stop it, use these exceptions... Might be also a good idea to mention that these can be used in libraries and not only in actor functions.

Reply:
I agree. Since this document is Best practices, it shouldn't be forcing anyone to do something. So "must" is too strong a word. Somebody could do error handling in their own way.
If you're gonna mention the libraries, mention both the actor's private library and shared library.

Reply:
> Somebody could do error handling in their own way.
Which I would discourage if it's not necessary for some reason. Having a coherent codebase is better than too much variation.

But otherwise I agree with the discussion here.